drydock/templates/kustomized/tutor13/extensions/kustomization.yml

```diff
@@ -30,3 +30,17 @@ configMapGenerator:
     labels:
         app.kubernetes.io/name: openedx
 {% endif -%}
+
+{% if not RUN_MYSQL -%}
+# MySQL job deletion based on the thread in https://github.com/kubernetes-sigs/kustomize/issues/4526
+patches:
+- target:
+    kind: Job
+    name: .*mysql.*
+  patch: |-
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: does-not-matter
+    $patch: delete
+{%- endif %}
```

Comment:
Is the notest MySQL job handled elsewhere?

Reply:
It is handled right here since the patch is using a regex to ignore all jobs containing the "mysql" word, it can be checked [here](https://github.com/eduNEXT/drydock/blob/main/drydock/templates/kustomized/tutor13/extensions/drydock-jobs/notes.yml#L5)

Reply:
At this point, isn't it easier to add `{% if not RUN_MYSQL -%}` to the jobs instead of applying patches at this level?. I see the patches more as last mile modifications.

Reply:
This is what I wanted to prevent considering few jobs could join the list. However, my approach could be considered a bit of magic since the jobs are actually written in the manifests but ignored by the patch, a fact that could lead to confusion.

Reply:
The issue with if not RUN_MYSQL is adding more jobs in the future that would need to specify this extra condition

Reply:
I already added labels to drydock jobs and changed the patch to filter based on such labels @MoisesGSalas @mariajgrimaldi @MAAngamarca